staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -498,8 +503,76 @@ func GetFieldManagerFlag(cmd *cobra.Command) string {
 	return GetFlagString(cmd, "field-manager")
 }
 
+type DryRunStrategy int
+
+const (
+	DryRunNone DryRunStrategy = iota
+	DryRunClient
+	DryRunServer
+)
+
+func (d DryRunStrategy) Client() bool {
+	return d == DryRunClient
+}
+
+func (d DryRunStrategy) Server() bool {
+	return d == DryRunServer
+}
+
+func (d DryRunStrategy) None() bool {
+	return d == DryRunNone
+}
```

Comment:
I don't think these functions have a lot of value

Reply:
Removed